scripting_guide.txt

```diff
@@ -0,0 +1,62 @@
+basic sintax elements
+	escape sequences
+		$$ escapes the $ simbol
+		${ escapes {
+		$} escapes }
+		// is used to write comments in the scripts
+	logic operators:
+		standard logic operators such as && || ^ ! are available, in absence op parentesis they follow the order they are written in the file
+		> >= < <= 
+	Math operators: 
+		standard math operators such as + - * / % are available, in absence op parentesis they follow the order they are written in the file
+	( and ) behave like most other languages
+	
+
+variables are by default string values. you can create flag or integer variables using $flag[name] or $int[name]
```

Comment:
Do we need to define `bools` and `ints` here? I think strings are OK since we can compare against both these types in strings in JavaScript and keeps the syntax smaller uwu